reservacionVuelo/Screens/SignUp.js

```diff
@@ -1,16 +1,111 @@
 import React, { Component } from 'react';
-import { View } from 'react-native';
+import { StyleSheet, Text, TextInput, TouchableHighlight, View } from 'react-native';
+import CheckBox from '@react-native-community/checkbox';
+import Icon from 'react-native-vector-icons/MaterialCommunityIcons';
 
 export class SignUp extends Component {
     constructor() {
-
+        super()
     }
 
     render() {
         return (
-            <View>
+            <View style={styles.mainContainer} >
+                <Text style={styles.titleStyle}>Sign Up</Text>
+                {/**TextInputs */}
+                <View style={styles.textInputSpacing}>
+                    <Text style={styles.textInputTitle}>First Name</Text>
+                    <TextInput style={styles.textInputStyle}></TextInput>
+                </View>
+                <View style={styles.textInputSpacing}>
+                    <Text style={styles.textInputTitle}>E-mail *</Text>
+                    <TextInput style={styles.textInputStyle}></TextInput>
+                </View>
+                <View style={styles.textInputSpacing}>
+                    <Text style={styles.textInputTitle}>Password *</Text>
+                    <TextInput style={styles.textInputStyle} secureTextEntry={true}></TextInput>
```

Comment:
**As for the design, it's very good!**

I only recommend using an indentation of 2 and also running ESLint on your files, as it will help you a lot with this type of problem, semicolons, commas, spaces, etc.

Here is the documentation:
- **Page**: https://eslint.org/docs/user-guide/getting-started
-  **Extension in vscode**: https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint